projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+
+
+frontend_publish_test_image:
+  image: docker:19.03.15
+  variables:
+    DOCKER_DRIVER: overlay2
+    DOCKER_TLS_CERTDIR: ''
+    DOCKER_HOST: tcp://localhost:2375
+  services:
+    - docker:19.03.15-dind
```

Comment:
nit: you can extract as yaml anchor

Reply:
Does that work when across different files?

Reply:
It might depending on the order of the includes but that's not what I meant. 
I think each project needs to stand on its own. 

I meant to declare it in this file ... 

Though make sure to add a prefix frontend_images or something to avoid conflicts.

Reply:
Yes, no probem.